packages/paste-core/layout/stack/__tests__/stack.test.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import {render as testRender} from '@testing-library/react';
+import {Card} from '@twilio-paste/card';
+import {Theme} from '@twilio-paste/theme';
+import {getStackDisplay, Stack} from '../src';
+
+describe('Stack Unit Tests', () => {
```

Comment:
Would be worth having tests for `getStackChildMargins` and `getStackStyles` too, I suspect